app/src/main/java/org/mozilla/fenix/settings/address/interactor/AddressEditorInteractor.kt

```diff
@@ -25,6 +26,22 @@ interface AddressEditorInteractor {
      * @param addressFields A [UpdatableAddressFields] record to add.
      */
     fun onSaveAddress(addressFields: UpdatableAddressFields)
+
+    /**
+     * Deletes the provided address from the autofill storage. Called when a user
+     * taps on the save menu item or "Save" button.
+     *
+     * @param address An [Address] record to delete.
```

Comment:
Wrong @param name and message - this is a `guid`.

Reply:
I originally passed the address but forgot to change this when switching to guid.